mne/annotations.py

```diff
@@ -431,11 +457,35 @@ def _write_annotations(fid, annotations):
     end_block(fid, FIFF.FIFFB_MNE_ANNOTATIONS)
 
 
+def _write_annotations_csv(fname, annot):
+    pd = _check_pandas_installed(strict=True)
+    meas_date = _handle_meas_date(annot.orig_time)
+    dt = datetime.utcfromtimestamp(meas_date)
+    onsets_dt = [dt + timedelta(seconds=o) for o in annot.onset]
+    df = pd.DataFrame(dict(onset=onsets_dt, duration=annot.duration,
+                           description=annot.description))
+    df.to_csv(fname, index=False)
+
+
+def _write_annotations_txt(fname, annot):
+    content  = "# MNE-Annotations\n"
+    if annot.orig_time is not None:
+        meas_date = _handle_meas_date(annot.orig_time)
+        orig_dt = datetime.utcfromtimestamp(meas_date)
+        content += "# orig_time : %s   \n" % orig_dt
+    content += "# onset, duration, description\n"
```

Comment:
It is fairly standard to have a header line with the column headers in the first line. All readers should support this header line, so I'd remove the comment here.

Reply:
I don't understand your comment. Are you proposing this:

```diff
-    content += "# onset, duration, description\n"
+    content += "onset, duration, description\n"
```
This complicates the parsing of the `.txt` using `np.loadtxt` quite a lot. And by the end if anyone loads a `.txt` should be able to see it anyway.

I'm not sure about this change. Anyone else?

Reply:
`skiprows=1`

Reply:
@cbrnr what you want is a csv file you can read with pandas. It's not anymore a txt file you can read with np.loadtxt or matlab without understanding the file specifics.

I would therefore stick to the current proposal. Any other opinion anyone?

Reply:
No idea which is more standard. It would be good to check if pandas still works without the `#` on the first line. If it does not, then having `#` on the first line is probably worth it even if it's less standard. (On readers that can support a no-#-first-line-header, the data fields will presumably just appear as "column 1/2/3" or something similar, which seems less than ideal but livable.)

Reply:
with the # you'll need to specify column names on read. It will treat the # as a comment and will ignore it as np.loadtxt or a matlad reader of floats in txt files

Reply:
Yes, basically I wanted a CSV file but without the timestamps. The idea was to create a file that can be easily read by the user, and this means `pd.read_csv` and not `np.loadtxt` for me. Either way, I can live with the #, which means I have to manually specify the column names.